### PR TITLE
Add styling

### PR DESCRIPTION
**Pull request by hamidazim321**
## PR Summary
- I have added all the stylings according to the template provided by Microverse
- I have added the exact font stylings in the book form input from the template as required by last reviewer
## For reviewers
- Note the content in some of the areas such as `completed`% and `Current chapter` is different from the template, this is because the data is for each book is fetched from API.

### CODE REVIEW

File: src/styles/BookForm.css [hunk 1]
```diff
@@ -36,6 +36,12 @@ form {
 .select-category,
 button {
   padding: 0.8rem;
+  font-family: sans-serif;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
 }
```
Comment (DammyShittu) [inline src/styles/BookForm.css:48]: - [ ]  We should replicate the design exactly. That includes the font used. Kindly use the font styling we have on the design.

  image

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @hamidazim321,

## Highlights
✔️ Page layout and design

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/BookForm.css [hunk 2]
```diff
@@ -16,7 +16,7 @@
```
Comment (DammyShittu) [inline src/styles/BookForm.css:16]: - [ ] Kindly note that the previous review meant your entire app fonts, not just a button. Kindly compare the font of different parts of your app, and that of the design and fix the fonts. You can see a lot of differences in the text fonts, like the current chapter part, Hunger games part, button font, chapter 17, the Bookstore CMS and the menu links and so on. The fonts should all match.

  Your Implementation | Zeplin Design
  :---:|:---:
  odde | image

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @hamidazim321,

## Highlights
✔️ Page styling

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @hamidazim321 , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Awesome job using the right fonts the match the original design. ✔
- The milestone requirements have been met with success. 🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**